src/status_im2/contexts/chat/messages/avatar/view.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im2.contexts.chat.messages.avatar.view
+  (:require [utils.re-frame :as rf]
+            [react-native.core :as rn]
+            [quo2.core :as quo]
+            [status-im2.contexts.chat.messages.avatar.style :as style]))
+
+(defn avatar
+  [from size]
```

Comment:
perhaps`contact` would be better in place of "from"? it seems "from" is quite coupled to the use case you have here?

Reply:
oh yes, `public-key` probably better?